### ISSUE TITLE
Unable to delete external domains

### ISSUE THREAD

**Issue by huv1k** [vercel/vercel#1739]
Its impossible to delete external domains. `now domain rm DOMAIN` just throws error:
```
 Error! An unexpected error occurred in domain: TypeError: Cannot read property 'length' of undefined
    at module.exports.t.default.t.--yes.r (/snapshot/repo/dist/index.js:1:406154)
    at new Promise (<anonymous>)
    at confirmDomainRemove (/snapshot/repo/dist/index.js:1:405922)
    at rm (/snapshot/repo/dist/index.js:1:406615)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
`now-cli: 12.1.11` `OSX Mojave 10.14.1`

**Comment by revskill10** [comment]
I got the same error today.

**Comment by goughjt** [comment]
This was fixed for me by upgrading to `now-cli@12.1.12`?
I upgraded like this:
`$ yarn global upgrade now@12.1.12`

**Comment by juliangruber** [comment]
@goughjt got it
